src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -216,6 +216,7 @@ internal enum MessageID
         IDS_FeatureVarianceSafetyForStaticInterfaceMembers = MessageBase + 12791,
         IDS_FeatureConstantInterpolatedStrings = MessageBase + 12792,
         IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction = MessageBase + 12793,
+        IDS_FeatureRecordStructs = MessageBase + 12794,
```

Comment:
:'(

Reply:
What's the matter?

Reply:
![it ok](https://thumbs.gfycat.com/SerpentineWaryAgama-size_restricted.gif)

Reply:
that's hte one i was using :(

Reply:
Don't worry, we'll have all sort of ID conflicts, but they'll all be easy to solve once we get to that. Most likely your feature will go in first and I'll hit the conflict ;-)